src/components/BottomModal.js

```diff
@@ -11,6 +11,8 @@ import ButtonUseTouchable from "../context/ButtonUseTouchable";
 import getWindowDimensions from "../logic/getWindowDimensions";
 import DebugRejectSwitch from "./DebugRejectSwitch";
 
+const isModalOpenedref = createRef();
```

Comment:
there is no need for this to be a ref, it can just be a `let isModalOpened` global variable.